src/settings/PathConfigStorage.cpp

```diff
@@ -163,6 +222,65 @@ PathConfigStorage::PathConfigStorage(const SliceDataStorage& storage, int layer_
     }
 }
 
+const GCodePathConfig *PathConfigStorage::MeshPathConfigs::getInset0Config(int layer_nr) const
+{
+    return (layer_nr == 0)? &inset0_config_layer0 : &inset0_config;
+}
+
+const GCodePathConfig *PathConfigStorage::MeshPathConfigs::getInsetXConfig(int layer_nr) const
+{
+    return (layer_nr == 0)? &insetX_config_layer0 : &insetX_config;
+}
+
+const GCodePathConfig *PathConfigStorage::MeshPathConfigs::getSkinConfig(int layer_nr) const
```

Comment:
These three have a `layer_nr` parameter that's not marked as `const` while it is marked as `const` in the header file.